lib/buffer.js

```diff
@@ -401,7 +400,8 @@ Buffer.isBuffer = function isBuffer(b) {
 
 Buffer.compare = function compare(a, b) {
   if (!isUint8Array(a) || !isUint8Array(b)) {
-    throw new TypeError('Arguments must be Buffers or Uint8Arrays');
+    throw new errors.TypeError('ERR_INVALID_ARG_TYPE', 'arguments',
```

Comment:
IMHO it should be `buf1` and `buf2` alike in the API docs https://nodejs.org/api/buffer.html#buffer_class_method_buffer_compare_buf1_buf2